build/extension_manifest.json

```diff
@@ -22,19 +22,22 @@
 			"*://de.wikipedia.org/*",
 			"*://eu.wikipedia.org/*",
 			"*://tr.wikipedia.org/*",
-			"*://es.wikipedia.org/*"
+			"*://es.wikipedia.org/*",
+			"*://fr.wikipedia.org/*"
 		],
 		"exclude_globs": [
 			"*://en.wikipedia.org/wiki/Special:*",
 			"*://de.wikipedia.org/wiki/Spezial:*",
 			"*://eu.wikipedia.org/wiki/Berezi:*",
 			"*://tr.wikipedia.org/wiki/%C3%96zel:*",
+			"*://fr.wikipedia.org/wiki/Spécial:*",
 			"*://es.wikipedia.org/wiki/Especial:*",
 			"*://en.wikipedia.org/*title=Special:*",
 			"*://de.wikipedia.org/*title=Spezial:*",
 			"*://eu.wikipedia.org/*title=Berezi:*",
 			"*://tr.wikipedia.org/*title=%C3%96zel:*",
-			"*://es.wikipedia.org/*title=Especial:*"
+			"*://es.wikipedia.org/*title=Especial:*",
+			"*://fr.wikipedia.org/*title=Spécial:*"
```

Comment:
Does it understand `Spécial` or do we need to surround it with URL friendly characters `*title=Sp%C3%A9cial`?

Reply:
Nice catch, Harumi! In fact it does need to be percent-encoded. PR updated. Thanks :)